prow/cluster/jobs/all-periodics.yaml

```diff
@@ -279,8 +279,58 @@ periodics:
           - entrypoint
           - prow/perf_benchmark_test.sh
 
-- cron: "0 18 * * *"  # starts every day at 18:00PM UTC
-  name: istio-upgrade-helm-chart
+# upgrade using helm from 1.2.8 to 1.3.4
+- cron: "0 2 * * *"  # starts every day at 02:00AM UTC
+  name: istio-upgrade-using-helm
```

Comment:
job names need to be unique, should probably include from and to in the name

Reply:
oh, I see.